includes/data-stores/class-wc-admin-reports-coupons-data-store.php

```diff
@@ -249,4 +258,48 @@ protected function get_cache_key( $params ) {
 		return 'woocommerce_' . self::TABLE_NAME . '_' . md5( wp_json_encode( $params ) );
 	}
 
+	/**
+	 * Make an entry in the wc_order_coupon_lookup table for an order.
```

Comment:
Should this be "Make or Update an entry"

Reply:
Agreed and updated!